tests/test_htmIndex.py

```diff
@@ -308,6 +340,29 @@ def testDefaultFilterAndFilterMap(self):
                 self.assertTrue(aprimeFluxFieldName in lcat.refCat.schema)
                 break  # just need one test
 
+    def testProperMotion(self):
+        """Test proper motion correction"""
+        center = make_coord(93.0, -90.0)
+        loader = LoadIndexedReferenceObjectsTask(butler=self.test_butler)
+        references = loader.loadSkyCircle(center, self.search_radius, filterName='a').refCat
+        original = references.copy(True)
+
+        # Zero epoch change --> no proper motion correction (except minor numerical effects)
+        loader.applyProperMotions(references, self.mjd)
+        self.assertFloatsAlmostEqual(references["coord_ra"], original["coord_ra"], rtol=1.0e-14)
+        self.assertFloatsAlmostEqual(references["coord_dec"], original["coord_dec"], rtol=1.0e-14)
+        self.assertFloatsEqual(references["coord_ra_err"], original["coord_ra_err"])
+        self.assertFloatsEqual(references["coord_dec_err"], original["coord_dec_err"])
+
+        # One year difference --> coordinate difference should be self.proper_motion in RA only
+        loader.applyProperMotions(references, self.mjd + 365.25)
+        for orig, ref in zip(original, references):
+            self.assertFloatsAlmostEqual(orig.getCoord().bearingTo(ref.getCoord()).asDegrees(),
+                                         180.0, rtol=1.0e-9)
+            self.assertFloatsAlmostEqual(orig.getCoord().separation(ref.getCoord()).asArcseconds(),
+                                         self.proper_motion, rtol=1.0e-12)
```

Comment:
I'm somewhat surprised the tolerance has to be that high for both direction and distance.

I was about to say you should use `assertSpherePointListAlmostEqual`, until I thought about the nature of the test (move X along RA only). So, maybe a comment about why the above test method wouldn't be ideal?

Reply:
I believe this tolerance was based on writing out rather few digits to to the "original catalog" that is ingested. I have overhauled the code to compare angles and use absolute tolerance. Here is the new code:
```
self.assertAnglesAlmostEqual(orig.getCoord().separation(ref.getCoord()),
                             self.proper_motion_amt, maxDiff=1.0e-6*lsst.geom.arcseconds)
self.assertAnglesAlmostEqual(orig.getCoord().bearingTo(ref.getCoord()),
                             self.proper_motion_dir, maxDiff=1.0e-4*lsst.geom.arcseconds)
```
the separation is quite a bit better than this, but I didn't want to overdo it. Direction is harder to compute accurately, but the allowed error is quite acceptable.

Reply:
Shouldn't be using `under_scores` in variable names.

Reply:
Good point. I'll fix it (though I don't think I made up those names; didn't you come up with them?)

Reply:
I don't think so. Probably they came in with the origin of the "LSST format catalogs", which I think was back when the policy was unclear and thought to be in transition.